query-planner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "apollo-query-planner"
-version = "0.0.3"
+version = "0.0.4"
```

Comment:
I think version changes should wait for their actual release. I wasn't sure about this until I thought about it this way: multiple feature additions still end up on the same minor, we only bump the minor once on release. The same could be said for bug fixes, though we'll likely release a patch for most bug fixes individually.

Reply:
I agree, in this case though, this is because I forgot updating this one yesterday (Assuming we want to keep query-planner-wasm and query-planner the same.